README.md

```diff
@@ -28,7 +28,7 @@
 
 **Functional tools that couldn’t be simpler.**
 
-We’re proud to present *1-liners* – a dead simple functional utility belt. **[103 one-liner functions][docs]** (and counting). Each hand-crafted with love and attention.
+We’re proud to present *1-liners* – a dead simple functional utility belt. **[113 one-liner functions][docs]** (and counting). Each hand-crafted with love and attention.
```

Comment:
It's automated.
https://github.com/1-liners/1-liners/blob/master/package.json#L20